docs/operation/volume_workflow/volume_creation_deletion_cli.rst

```diff
@@ -0,0 +1,72 @@
+Volume Workflow CLI Operations
+==============================
+To use persistent volumes within a MetalK8s cluster, a volume object needs
+to be created. Volume objects depend on StorageClasses to function properly.
+
+Volume Creation CLI
+-------------------
+This section highlights how to create volume objects in a MetalK8s cluster
+using the **CLI**.
+
+.. important::
+   Before performing any of the volume creation steps below, ensure that
+   you have completed the storageclass creation procedure listed in
+   :doc:`/operation/volume_workflow/storageclass_creation`
+
+#. Create a sample volume.
+
+   Save the volume definition file as **volume.yml**.
+
+   .. code-block:: shell
+
+       apiVersion: storage.metalk8s.scality.com/v1alpha1
+       kind: Volume
+       metadata:
+         name: <volume_name>
+       spec:
+         nodeName: <node_name>
+         storageClassName: <storageclass_name>
+         rawBlockDevice:
+           devicePath: <device_path>
+
+#. Apply the volume.
```

Comment:
Create the volume

Title don't have a period, they're not sentence.

Reply:
Hmmm, this was not a title... I see it as a number `listing`